### DISCUSSION TITLE
Advanced Schedulers

### DISCUSSION THREAD

**Discussion by rugved-bodke-stigya** [saleweaver/python-amazon-sp-api#1413] [Ideas]
To check the integrity of data(attributes/images) on Catalog and Listing, If any attributes are mismatching then should be reported or we can take actions to correct them

**Comment by saleweaver** [comment]
Look at Notifications, this feature is available.

**Reply by rugved-bodke-stigya** [reply]
Notifications are only available for ITEM_PRODUCT_TYPE_CHANGE, BRANDED_ITEM_CONTENT_CHANGE, this covers usually 5-10 attributes, I was looking at full attributes list.

Anyways thanks though, I have created SQS and redirected them to slack through Lambda Functions

![image](https://github.com/saleweaver/python-amazon-sp-api/assets/156916874/5bcb08b4-fc8b-4be6-bf53-9e1dfaac42e8)
